friends/src/components/FriendsList.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+export class FriendsList extends Component {
+	render() {
+		return (
+			<div>
+				<h1>My Friends!</h1>
+				{this.props.friends.map(friend => {
+					return (
+						<h6>
+							My name is {friend.name} I am {friend.age} years old. My email is {friend.email} and my ID
+							is {friend.id}.
+						</h6>
```

Comment:
You're missing key. Don't forget to check your terminal and console for any errors.